frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/common.ts

```diff
@@ -50,6 +51,7 @@ export const initializeCommonMetadata = (
   if (template) {
     entity.addLabel(LABEL_USED_TEMPLATE_NAME, getName(template));
     entity.addLabel(LABEL_USED_TEMPLATE_NAMESPACE, getNamespace(template));
+    entity.addAnotation(VALIDATIONS, getAnnotations(template)?.[VALIDATIONS]);
```

Comment:
can you please make a new function called initializeCommonTemplateMetadata and put it there instead? 
This function is also used for creating VMs and we are not really expecting VMs to include validations.